src/diagnose.ts

```diff
@@ -273,6 +276,20 @@ export class DiagnoseTool {
     )
   }
 
+  private getCustomClientFilePath() {
+    const flagIndex = process.argv.indexOf("--config")
+
+    if (flagIndex !== -1 && flagIndex + 1 < process.argv.length) {
+      const filePath = process.argv[flagIndex + 1]
+
+      if (!path.isAbsolute(filePath)) {
+        return filePath
+      } else {
+        return path.resolve(`../${filePath}`)
+      }
```

Comment:
```suggestion
      if (path.isAbsolute(filePath)) {
        return filePath
      } else {
        return path.resolve(filePath)
      }
```

If if-statement was in reverse: it making an absolute path even more absolute, and it not making a relative path an absolute path. I reversed the if-statement and made a change for the absolute path:

I'm not entirely sure why locally I had to specify `--config ../my_appsignal.cjs` for a custom relative file path, but here we don't. I guess because it's executed within the path of the script. But this change works.

Reply:
I could make it work in Docker using the redis-express app. I'll change it.